docs/examples/queries/tournaments-by-videogame.md

```diff
@@ -8,7 +8,7 @@ In others words, "Give me the next _x_ upcoming tournaments for _Videogame(s)_."
 The first example will be for a single videogame,
 and the second example will be for an array of videogames.
 
-For now, you can view the mapping of videogame IDs to their names <a href="https://docs.google.com/spreadsheets/d/1l-mcho90yDq4TWD-Y9A22oqFXGo8-gBDJP0eTmRpTaQ/edit?usp=sharing" target="_blank">in this sheet here.</a>
+You can find the ID for a given game via the instructions in the [get game id by name example query](/videogame-id-by-name).
```

Comment:
this is a broken link. i thought it might not work so i checked on the preview build for this PR:
https://deploy-preview-166--smashgg-developer-portal.netlify.app/docs/examples/queries/tournaments-by-videogame

any PR on this repo should trigger a preview build that will allow you to preview the changes in your browser. you can also build the doc site locally, there should be instructions here (spoiler: its super simple): https://github.com/smashgg/developer-portal/tree/master/website